modules/migrating-to-multi-arch-cli.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * updating/migrating-to-multi-payload.adoc
+
+:_content-type: PROCEDURE
+[id="migrating-to-multi-arch-cli_{context}"]
+= Migrating clusters to a multi-arch/heterogenous payload using the CLI
+
+.Prerequisites
+
+* You must be logged in as a user with `cluster-admin` privileges on all clusters.
+
+.Procedure
+
+. Install your {product-title} cluster on your preferred platform. For more information on selecting a supported platform for your cluster installation, see _Selecting a cluster installation type_ in the Additional resources section.
+. Set the channel to candidate-4.13 with `oc adm upgrade channel candidate-4.13`. For more information about channels, refer to _Understanding update channels and releases_ listed in the Additional resources section.
++
+[source,terminal]
+----
+$ oc adm upgrade channel candidate-4.13
+----
+
+. Upgrade to multi-arch with the `oc adm upgrade --to-multi-arch` command:
++
+[source,terminal]
+----
+oc adm upgrade --to-multi-arch.
```

Comment:
The snippet above had a `$` prompt, this one does not

Reply:
good catch! ty :)